src/praisonai/praisonai/endpoints/providers/openai_compat.py

```
@@ -0,0 +1,521 @@
+                return self._handle_completions(input_data)
+            elif name == "models":
+                return self._handle_models()
+            elif name == "tools_invoke":
+                return self._handle_tools_invoke(input_data)
+            else:
+                return InvokeResult(
+                    ok=False,
+                    status="not_found",
+                    error=f"Unknown endpoint: {name}",
+                )
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)